code/Render/Vulkan/RenderViewVk.cpp

```diff
@@ -525,6 +525,20 @@ void RenderViewVk::setViewport(const Viewport& viewport)
 	vkCmdSetViewport(*frame.graphicsCommandBuffer, 0, 1, &vp);
 }
 
+void RenderViewVk::setScissor(const Rectangle& scissor)
+{
+	T_ASSERT(scissorRect.left >= 0);
+	T_ASSERT(scissorRect.top >= 0);
+	T_ASSERT(scissorRect.width >= 0);
+	T_ASSERT(scissorRect.height >= 0);
+
+	const auto& frame = m_frames[m_currentImageIndex];
+	VkRect2D vkScissor = {};
+	vkScissor.offset = VkOffset2D { scissor.left, scissor.top };
+	vkScissor.extent = VkExtent2D { static_cast<uint32_t>(scissor.width), static_cast<uint32_t>(scissor.height) };
```

Comment:
Not using static_cast for primitive types since it's too verbose, sticking to C-style casts for those.